MemberExpectations.md

```diff
@@ -0,0 +1,60 @@
+if the results run counter to their person point of view.  A member who does
+not support a team decision or current team value may express the member's
+opposition within the team in an appropriate manner, but must be extremely
+careful when expressing the opinion externally.  External communication must
+not be done with the intent of undermining a decision, project values, or
+actions of the team. If another team member expresses that they believe external
+communications appear to have the intent to undermine the committee
+it is the responsibility of the individual expressing their opinions to rectify
+the situation immediately. If a member feels that there is no recourse aside
+from going public to campain against the teams decision on an extremely
+contentious issue then they should resign from the team before doing so.
+
+Members of our leadership groups must also conduct themselves in a
+professional and respectful manner. Some general guidelines include:
+
+- Serve as ambassadors of the vision, mission and operating
+  principles of the Node.js project.
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes - however they should act swiftly to
+  acknowledge mistakes and correct them. Most often this can mitigate
+  any damage done.
+- Aim to remediate first and then discuss.  If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
+- Treat all community members with respect, consideration, and highest
+  standards of ethical conduct.  
+- Valuing a diversity of views and opinions. Strive to avoid preferential
+  treatment, and hold everyone (including ourselves) accountable to the same
+  set of standards.  Everyone gets to speak up. 
+- Deal with issues directly with the person in question. Resist complaining
+  about others in the project in a public sphere.
+- Build trust by keeping your promises.
+- Be the model of accountability and leadership. Provide the example of
+  ownership and stewardship that everyone can follow to success.
+- Commit to ongoing development and learning best practices for governing.
+- Critique ideas rather than individuals, discussing any concerns in person
+  whenever possible, and taking responsibility for our statements by
+  speaking as much as possible
+  in the first person (.I. statements) rather than in the third person.
+  
+Finally, the public behavior of members reflects on the Node.js project.  It is
+often difficult for those external to the project to separate actions which 
+reflect the members private views from how they will act within
+the project.  Members are expected to avoid acting in a way that might bring
+disrepute to the Node.js organization and avoid any actions that might be
+interpreted as undermining the core values of the project.
```

Comment:
Here is an attempt at some alternate language for the above paragraph

> While the guidelines above are primarily focusing on the spaces where we participate
in official foundation work (github, irc, conferences), it is important to recognize
that the public behavior of members reflects on the Node.js project. Members are
expected to hold each other accountable in external spaces and approach external
situations with the same candor and respect as we do internally. This includes
remediating first and discussing after.

Reply:
Will incorporate in next pass through.